ui/client/components/right-panel/UserRightPanel.js

```diff
@@ -65,9 +66,12 @@ class UserRightPanel extends Component {
         <SpinnerWrapper isReady={isReady}>
 
           <Scrollbars renderThumbVertical={props => <div {...props} className="thumbVertical"/>} hideTracksWhenNotNeeded={true}>
-            
-            <ProcessState process={fetchedProcessDetails} processState={processState} isStateLoaded={isStateLoaded}/>
-            
+
+            { fetchedProcessDetails.isArchived || fetchedProcessDetails.isSubprocess ?
+              <ProcessInfo process={fetchedProcessDetails}/> :
```

Comment:
it's kind of misleading when we have ProcessInfo only for subprocess/archive... I think it would look better if we put this logic inside ProcessState/StateIcon - they look more or less the same anyway?

Reply:
Hmmm... Okey, I will change it to ProcessInfo component.

Reply:
Done